autopilot/autopilot.go

```diff
@@ -163,12 +165,17 @@ func (ap *Autopilot) Handler() http.Handler {
 	return jape.Mux(tracing.TracedRoutes(api.DefaultAutopilotID, map[string]jape.Handler{
 		"GET    /alerts":         ap.handleGETAlerts,
 		"POST   /alerts/dismiss": ap.handlePOSTAlertsDismiss,
-		"GET    /config":         ap.configHandlerGET,
-		"PUT    /config":         ap.configHandlerPUT,
-		"POST   /debug/trigger":  ap.triggerHandlerPOST,
-		"POST   /hosts":          ap.hostsHandlerPOST,
-		"GET    /host/:hostKey":  ap.hostHandlerGET,
-		"GET    /status":         ap.statusHandlerGET,
+
+		"GET    /webhooks":    ap.handleAlertsWebhooksGET,
```

Comment:
`nit` if we move the `webhooks` routes to the bottom we preserve alphabetic ordering